File: otimizacao/src/main/java/com/otimizacao/domain/MixDeProducao.java
package com.otimizacao.domain;

import java.util.List;

public class MixDeProducao {
	
	private FuncaoObjetiva funcaoObjetiva;
	private List<Restricao> restricoes;
	
	public FuncaoObjetiva getFuncaoObjetiva() {
		return funcaoObjetiva;
	}
	public void setFuncaoObjetiva(FuncaoObjetiva funcaoObjetiva) {
		this.funcaoObjetiva = funcaoObjetiva;
	}
	public List<Restricao> getRestricoes() {
		return restricoes;
	}
	public void setRestricoes(List<Restricao> restricoes) {
		this.restricoes = restricoes;
	}

}

File: client/src/app/modulos/otimizacao/domain/mix-de-producao.ts
import { Restricao } from './restricao';
import { FuncaoObjetiva } from './funcao-objetiva';

export class MixDeProducao {

  funcaoObjetiva: FuncaoObjetiva;
  restricoes: Restricao[];

  constructor() {
    this.funcaoObjetiva = new FuncaoObjetiva();
    this.restricoes = [];
  }
}